demos/constraint/ConstrainedPlanningCommon.py

```diff
@@ -254,6 +262,10 @@ def solveOnce(self, output=False, name="ompl"):
         return stat
 
     def setupBenchmark(self, planners, problem):
+        if ot:
+            print("Benchmarking not available, no ompl.tools")
+            return
```

Comment:
```suggestion
            exit(0)
```
If you don't exit, run_benchmark will still get called, which produces an error.